server.js

```diff
@@ -1,33 +1,59 @@
 import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
+import athletesData from './data/athletes-data.json'
+
+console.log(athletesData.length)
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
 const port = process.env.PORT || 8080
 const app = express()
-
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+//Down below routes are defined
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Hello world, hej hej')
+})
+//Down below routes are defined
+// //http://localhost:8080/athletes
+// app.get('/athletes', (req, res) => {
+//   res.json(athletesData)
+// })
+
+app.get('/athletes/:id', (req, res) => {
+  const showId = req.params.id 
+  console.log(showId)
+  const show = athletesData.filter(item => +item.competitorid === +showId)
```

Comment:
I think this would be better with `.find` so that it returns a single object, rather than an array like `.filter` does.